API Docs Generation doesn't work

The generation of API documentation with `npm run docusaurus gen-api-docs all` doesn't work and currently results in the following error:
```
OpenAPI docs plugin ID must be specified when more than one plugin instance exists.
```

This is caused by multiple occurrences of the `docusaurus-plugin-remote-content`, e.g.:
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/docusaurus.config.js#L62
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/docusaurus.config.js#L73

They should be merged into one to solve the problem.

It can't work since there are still three problems in the config file:
1. `/openApi/bpdm/pool.yaml` and `/openApi/bpdm/pool.yaml` doesn't exist
2. The `Template Kit` folder doesn't exist
3. The traceability open-apis are missing

Would be nice to know which commands you are using to get it working.

Steps to reproduce:
1. Check out the `main` branch
5. Run `npm run docusaurus gen-api-docs all`
6. Error:
```
Loading of api failed for "[...]/eclipse-tractusx.github.io/openApi/bpdm/pool.yaml"
[ERROR] Error: ENOENT: no such file or directory, lstat '[...]/eclipse-tractusx.github.io/openApi/bpdm/pool.yaml'

```

I fixed it in #232. Further, there are still Open API specifications missing. I addressed it in #233.

--- a/docusaurus.config.js
+++ b/docusaurus.config.js
@@ -97,26 +97,19 @@ const config = {
     [
       'docusaurus-plugin-openapi-docs',
       {
-        id: "bpdm-pool",
+        id: "api-docs",
         docsPluginId: "classic",
         config: {
-          bpdm: {
-            specPath: "./openApi/bpdm/pool.yaml",
-            outputDir: "./docs/kits/Business Partner Kit/Software Development View/Pool Api",
+          traceability_notifications: {
+            specPath: "./openApi/traceability/notifications.yaml",
+            outputDir: "./docs/kits/Traceability Kit/Software Development View/Notification API",
             sidebarOptions: {
               groupPathsBy: "tag",
             },
           },
-          irs: {
-            specPath: "./openApi/irs/irs-v1.0.yaml",
-            outputDir: "./docs/kits/Data Chain Kit/Software Development View/Job Api",
-            sidebarOptions: {
-              groupPathsBy: "tag",
-            },
-          },
-          notifications: {
-            specPath: "./openApi/notifications/notifications.yaml", // Path to designated spec file
-            outputDir: "./docs/kits/Template Kit/Software Development View/Notification API", // Output directory for generated .mdx docs
+          traceability_unique_id_push: {
+            specPath: "./openApi/traceability/unique-id-push.yaml",
+            outputDir: "./docs/kits/Traceability Kit/Software Development View/Unique ID Push API",
             sidebarOptions: {
               groupPathsBy: "tag",
             },
@@ -389,7 +382,7 @@ const config = {
             items: [
               {
                 html: `
-                  <img src='/img/EF_registered_wht_png.png'/>
+                  <img alt='' src='/img/EF_registered_wht_png.png'/>
                 `
               }
             ]

